src/client/theme-default/NotFound.vue

```diff
@@ -2,19 +2,19 @@
   <div class="theme">
     <h1>404</h1>
     <blockquote>{{ getMsg() }}</blockquote>
-    <a :href="$site.base" aria-label="go to home"> Take me home. </a>
+    <a :href="$site.base" aria-label="go to home">Take me home.</a>
   </div>
 </template>
 
-<script setup>
+<script setup lang="ts">
 const msgs = [
   `There's nothing here.`,
   `How did we get here?`,
   `That's a Four-Oh-Four.`,
   `Looks like we've got some broken links.`
 ]
 
-function getMsg() {
+const getMsg = () => {
```

Comment:
does this work without the change? I think having the `function` keyword here is valuable for readability

Reply:
It seemed it doesn't, we must declare function as `const` for it to be exposed. But let me double check 👀

Reply:
Oh sorry, it did work. I was mistaken by something 😅  Reverted the change!